### DISCUSSION TITLE
Status of hdt-fuseki

### DISCUSSION THREAD

**Discussion by namedgraph** [rdfhdt/hdt-java#176] [Q&A]
Is `hdt-fuseki` coming back? What are the issues prevent it?

**Comment by D063520** [comment]
@mielvds : I think you know something about this ....

**Comment by mielvds** [comment]
The issue was an incompatibility between Jena 4.X and the Fuseki 1 HDT implementation, which was outdated anyway since there's Fuseki 2 now.
There isn't much preventing a comeback except dev time and an understanding/modification of Fuseki 2's architecture :) Some info here: 

- https://github.com/rdfhdt/hdt-java/pull/144
- https://github.com/rdfhdt/hdt-java/pull/121
- https://issues.apache.org/jira/browse/JENA-2268
- https://lists.apache.org/thread/vhk3bjdfbsh8gqv6p7sdl4osp83ompvd

**Comment by namedgraph** [comment]
https://github.com/the-qa-company/qEndpoint
